src/commands/database/create.mjs

```diff
@@ -1,19 +1,58 @@
 //@ts-check
 
+import { FaunaError, fql } from "fauna";
+
 import { container } from "../../cli.mjs";
+import { commonQueryOptions } from "../../lib/command-helpers.mjs";
+import { throwForV10Error } from "../../lib/fauna.mjs";
 
-async function createDatabase() {
+async function createDatabase(argv) {
   const logger = container.resolve("logger");
-  logger.stdout(`TBD`);
+  const runV10Query = container.resolve("runV10Query");
+
+  try {
+    await runV10Query({
+      url: argv.url,
+      secret: argv.secret,
+      query: fql`Database.create({
+        name: ${argv.name},
+        protected: ${argv.protected ?? null},
+        typechecked: ${argv.typechecked ?? null},
+        priority: ${argv.priority ?? null},
+      })`,
+    });
+    logger.stdout(`Database ${argv.name} created`);
+  } catch (e) {
+    if (e instanceof FaunaError) {
+      throwForV10Error(e, {
+        onConstraintFailure: () =>
+          `Constraint failure: The database '${argv.name}' may already exists or one of the provided options may be invalid.`,
+      });
+    }
+    throw e;
+  }
 }
 
 function buildCreateCommand(yargs) {
   return yargs
     .options({
+      ...commonQueryOptions,
```

Comment:
the order these are registered in determines the order of the helpfile - thoughts on moving the common ones to the end?